pkgs/development/tools/language-servers/dcd/default.nix

```diff
@@ -0,0 +1,74 @@
+{ lib, stdenv, ldc, fetchFromGitHub }:
+
+let
+  dcdVersion = "0.15.2";
+  fetchSource = attrs: fetchFromGitHub (attrs // { name = attrs.repo; });
```

Comment:
IMO this indirection doesn't add much, and increases cognitive overhead reading the code.

Reply:
Removed it